tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_nonreturning_validation.py

```diff
@@ -208,3 +210,44 @@ def test_jumpf_in_nonreturning(eof_test: EOFTestFiller, first: bool, code_prefix
 def test_callf_to_nonreturning(eof_test: EOFTestFiller, container: Container):
     """Test EOF validation failing due to CALLF to non-returning section."""
     eof_test(container=container, expect_exception=EOFException.CALLF_TO_NON_RETURNING)
+
+
+VALID: List[Container] = [
+    Container(
+        name="returning_section_aborts_jumpf",
```

Comment:
If this is valid does it make sense to execute it?

Reply:
sorry, not sure what do you mean?

Reply:
Use `eof_state_test`.

Reply:
hm, I think it is `eof_state_test` which will execute it, so `eof_test` would be what you want